docs/source/start.ipynb

```diff
@@ -60,14 +60,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X['expiration_date'] = X['expiration_date'].apply(lambda x: '20{}-01-{}'.format(x.split(\"/\")[1], x.split(\"/\")[0]))\n",
-    "X[['lat', 'lng']] = X[['lat', 'lng']].astype('str')\n",
+    "X.ww['expiration_date'] = X['expiration_date'].apply(lambda x: '20{}-01-{}'.format(x.split(\"/\")[1], x.split(\"/\")[0]))\n",
```

Comment:
Ah ok. We provide some types to the `feature_types` arg to `infer_feature_types` below, but not for all of the columns. If we provided types for all the columns below, I'm guessing it would be fine to omit the usage of `ww` here, because we'd create the full schema in `infer_feature_types` no matter what.